docs/index.md

```diff
@@ -20,7 +20,6 @@ terraform {
   required_providers {
     wavefront = {
       source  = "vmware/wavefront"
-      version = "~> 5.0.5"
```

Comment:
I believe this should already allows the patches to float. So we don't technically need to update them every time. Might be good to ensure a major doesn't land without intent... Though we control the whole stack; so, that may be an unnecessary guard.

Reply:
My feeling was since this was a simple "hello world" example, best to keep the config to a minimum, and allow for the end user to add config as they see fit. Also, this kinda stuff has a strong tendency to fall way behind.

Reply:
Ah, I totally missed that this was in an example. 👍